data-models/Makefile

```diff
@@ -37,10 +37,56 @@ DOCKER_EXTRA_ARGS :=
 include ../make/common.mk
 include ../make/go.mk
 
+# Version of controller-gen to use for generating CRD deepcopy, client, etc.
+CONTROLLER_GEN_VERSION := v0.17.2
```

Comment:
v0.20.0 is latest, if we're going to pin we should pickup the newest one

Reply:
also, janitor has a controller-gen as well. It's using latest rather than a pinned version though. Checkout how the main makfile in the root dir is handling versions: https://github.com/NVIDIA/NVSentinel/blob/82e71803c9ea7efade4f715e2d11c453dda0e755/Makefile#L24 

We should probably put controller-gen there with that same pattern since we're using it in multiple modules now